### PR TITLE
🌱  Add dead link linter github workflow

### PR DESCRIPTION
**Pull request by furkatgofurov7**
**What this PR does / why we need it**:
Add dead link linter github workflow and fix some dead links. This github action fails on broken links when *.md files are modified.

### CODE REVIEW

File: .github/workflows/lint-docs.yaml
Comment (furkatgofurov7) [inline .github/workflows/lint-docs.yaml:6]: Note: this is for testing only, will be removed once it is agreed this change can be merged
/hold

**Review by lentzi90 — APPROVED** [review]
Nice!

File: .github/workflows/lint-docs.yaml
Comment (furkatgofurov7) [reply .github/workflows/lint-docs.yaml]: Removed it

**Comment by kashifest** [comment]
/approve
Squash the commits please

**Comment by furkatgofurov7** [comment]
> Squash the commits please

@kashifest I believe keeping is it better, since they are meant for different purposes (adding linter / fixing existing broken links)

**Comment by kashifest** [comment]
> > Squash the commits please
> 
> @kashifest I believe keeping it as is would be better, since they are meant for different purposes (adding linter / fixing existing broken links)

ok then lets keep it

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Pull request merged**
